ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -1487,6 +1487,8 @@ presubmits:
     cluster: api.ci
     context: ci/prow/e2e-libvirt
     decorate: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
```

Comment:
:+1:

Reply:
What does this mean exactly? Do we've any docs on these yaml files these days?

Reply:
this means when you have a release PR that changes this file or a template , this job will be rehearsed as part of the testing of that PR - something like that.  Until I set that, this PR never ran this job, and I needed it to.  Docs? :woman_shrugging:  probably :eyes:

Reply:
Thanks for explaining but my question was more like what does "rehearsal" means in here?

Reply:
/me will look for docs (if any)..

Reply:
found it :) https://github.com/openshift/release/blob/3fefbc5ab8e55408c943309f22736119318295a0/ci-operator/REHEARSALS.md